adm/src/main.rs

```diff
@@ -109,7 +109,9 @@ fn parse_args<'a>() -> ArgMatches<'a> {
             (about: "creates the genesis.batch file for initializing the validator")
             (@arg input_file:
              +takes_value ... "file or files containing batches to add to the resulting")
-            (@arg output: -o --output "choose the output file for GenesisData"))
+            (@arg output: -o --output "choose the output file for GenesisData")
+            (@arg ignore_required_settings: -i --ignore_required_settings
```

Comment:
I don't think there should be a short flag for this

Reply:
Fixed